lms/lib/courseware_search/test/test_lms_filter_generator.py

```diff
@@ -6,6 +6,7 @@
 from lms.lib.courseware_search.lms_filter_generator import LmsSearchFilterGenerator
 from common.djangoapps.student.models import CourseEnrollment
 from common.djangoapps.student.tests.factories import UserFactory
+from xmodule.course_block import CATALOG_VISIBILITY_ABOUT, CATALOG_VISIBILITY_NONE  # lint-amnesty, pylint: disable=wrong-import-order
```

Comment:
Don't just copy the lint amnesty ignore here, if there is no reason to put this import in the wrong order, it should just be ordered correctly.  If there is a reason to ignore the order, then we should add a comment as to why and add the pylint-disable without the `lint-amnesty` portion.